Makefile

```diff
@@ -84,6 +84,7 @@ OCI_REGISTRY ?= projects.registry.vmware.com/tanzu_framework
 
 .DEFAULT_GOAL:=help
 
+# TODO: Change package path to cli/runtime when this var is moved there.
 LD_FLAGS += -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/buildinfo.IsOfficialBuild=$(IS_OFFICIAL_BUILD)'
```

Comment:
This will not be moved to `cli/runtime` but was already moved to `cli/core`. @prkalle is this something that we need to keep or can this be removed?

Reply:
We need to keep this until all the plugins are using the package from `cli/core`. Last I checked, some of them were still using them directly from tanzu-framework (since the old vars are not removed).

Reply:
We copied this to `cli/core`, and left the old vars unchanged. I see only management-cluster plugin uses this variable to deploy telemetry job with different url for RTM vs eng builds.  So, If no other plugins uses this var (IS_OFFICIAL_BUILD), we can copy this var to `management-cluster` plugin and remove it from `cli/core` as well.
(we need to avoid plugins using any package from `cli/core`).